src/Components/TrainerApp/Registration/MultiStepForm.js

```diff
@@ -0,0 +1,176 @@
+            platform_trainer_enabled: this.state.platformTrainer,
+        }
+        try {
+            console.info(update);
+            var resp = await API.graphql(graphqlOperation(updatePlatformConfig, { input: update }));
+            this.setState({ platformConfig: resp.data.updatePlatformConfig });
+        } catch (error){
+            console.error("Failed to update platform configurations:", error);
+        }
+    }
+
+
+    handleChange = (event) => {
+
+        if (event.target.type === "checkbox") {
+            this.setState({ [event.target.id]: event.target.checked });
+        } else {
+            this.setState({ [event.target.name]: event.target.value })
+        }
+
+    }
+
+
+    render() {
+        const { step, user, platformConfig, firstName, lastName, email, phone, address, city, usstate, zip, logoFile, faviconFile, primaryColor, secondaryColor, customUrl, persWebEnabled, platformNutrition, platformTrainer } = this.state;
+        const inputValues = { user, platformConfig, firstName, lastName, email, phone, address, city, usstate, zip, logoFile, faviconFile, primaryColor, secondaryColor, customUrl, persWebEnabled, platformNutrition, platformTrainer };
+        <Progress />
+        switch (step) {
+            case 1:
+                return [
+                    <div key="registrationUser">
+                        <Progress step={step} />
+                        <UserDetails
+                            nextStep={this.nextStep}
+                            handleChange={this.handleChange}
+                            inputValues={inputValues}
+                            commit={this.commitUser}
+                        />
+                    </div>
+                ]
+            case 2:
+                return [
+                    <div key="registrationAccount">
+                        <Progress step={step} />
+                        <AccountDetails
+                            nextStep={this.nextStep}
+                            prevStep={this.prevStep}
+                            handleChange={this.handleChange}
+                            inputValues={inputValues}
+                        />
+                    </div>]
+            case 3:
+                return [
+                    <div key="registrationPlatform">
+                        <Progress step={step} />
+                        <PlatformDetails
+                            nextStep={this.nextStep}
+                            prevStep={this.prevStep}
+                            handleChange={this.handleChange}
+                            inputValues={inputValues}
+                            commit={this.commitPlatform}
+                        />
+                    </div>]
+            case 4:
+                this.finalCommit();
+                return [
+                    <div key="something made up">
+                        <Navigate to="/TrainerGettingStarted/" replace={true}/>
```

Comment:
@esam014 Just thought you might be interested. How I'm returning to the "home" page. Will need to update later, once we finalize the page structure, as well as probably should move to confirm button, but that can be done later. This works. 
Seems like an easy way to move around. Just notice the "/" at the beginning to force ref to root.